pkg/inngest/trigger.go

```diff
@@ -20,6 +24,36 @@ func (m MultipleTriggers) Triggers() []Trigger {
 	return m
 }
 
+func (m MultipleTriggers) Validate(ctx context.Context) error {
```

Comment:
Should we also ensure that each trigger contains either an event or cron? That'd be our fault, but it'd still be good to fail validation